housekeeping-vacuum-tool/README.md

```diff
@@ -1,34 +1,74 @@
-## Housekeeping Vacuum Tool
+# Housekeeping Vacuum Tool
 
-### Usage
+# Overview
 
-#### YAML Configuration
+The Housekeeping Vacuum Tool looks for any files and folders in the data locations of a Hive table that are not referenced in either the Hive metastore or the Housekeeping database. Any paths discovered are considered "orphaned data" and are then scheduled for removal via the Housekeeping process. This can be useful if a Housekeeping database has become corrupt and previous orphaned path locations have been lost. By running the Vacuum Tool against a Hive table one can identify data that is no longer referenced by Hive and is thus a candidate for deletion.  
```

Comment:
Yeah, I'm trying to avoid mentioning Circus Train here as Circus Train uses this but this shouldn't know anything about CT if you see what I mean? I'll try make the above more general so as not to be just about corrupt databases but also other situations where running HK like you describe fails.

Reply:
Yes, I see what you mean and it's fine. I agree that this tool should remain CT-free.